src/Code.ts

```diff
@@ -67,9 +67,10 @@ var QuestradeApiSession = function () {
             'method': 'get',
             'headers': this.authHeader
         };
+        var row = 1;
         for (var i = 0; i < this.accounts.length; i++) {
             var url = this.authData.api_server + 'v1/accounts/' + this.accounts[i]['number'] + '/positions';
-            writeJSONtoSheet(JSON.parse(UrlFetchApp.fetch(url, options).getContentText())['positions'], "Positions", this.accounts[i]['number'], i);
+            row = writeJSONtoSheet(JSON.parse(UrlFetchApp.fetch(url, options).getContentText())['positions'], "Positions", this.accounts[i], row);
```

Comment:
If my first account returns no positions, row is returned as undefined from the following line:
```
    if (keys.length < 1) {
        // Nothing to do, return
        console.error("Nothing to do, return");
        return;
    }
```
This causes the next account to pass in `currentRow = undefined`, and subsequent `getRange` calls will fail.

Reply:
This snippet of code should `return currentRow`

Reply:
I will merge this and post the fix after, since i don't think I can add commits to the PR